.ci.yaml

```diff
@@ -1265,77 +1265,41 @@ targets:
   - name: Linux_android complex_layout_android__compile
     recipe: devicelab/devicelab_drone
     presubmit: false
-    bringup: true
     timeout: 60
     properties:
       tags: >
         ["devicelab","android","linux"]
       task_name: complex_layout_android__compile
-      dependencies: >-
-        [
-          {"dependency": "open_jdk", "version": "11"}
-        ]
-      caches: >-
-        [
-          {"name": "openjdk", "path": "java11"}
-        ]
     scheduler: luci
 
   - name: Linux_android complex_layout_android__scroll_smoothness
     recipe: devicelab/devicelab_drone
     presubmit: false
-    bringup: true
```

Comment:
On the reland, should the mac changes get the same changes?

Reply:
Yes. They need it as well. 

Let me know when you can take a look at: https://github.com/flutter/flutter/pull/88319#issuecomment-899954954